account-kit/react/src/types.ts

```diff
@@ -1,4 +1,32 @@
 import type { UseMutationOptions } from "@tanstack/react-query";
+import type { ReactNode } from "react";
+import type { AuthType } from "./components/auth/types.js";
+
+export type AlchemyAccountsUIConfig = {
+  header?: ReactNode;
+  showSignInText?: boolean;
```

Comment:
these and sections still feel like they should probably be scoped further to just some `authModal` or similarly named prop. when we add the connect wallet button or connect wallet modal, these won't make as much sense as root props